test/OpenBCISample-test.js

```diff
@@ -55,9 +56,8 @@ describe('OpenBCISample',function() {
             //console.log(temp);
             var taco = new Buffer([0x81]);
             taco.copy(temp,2);
-            //console.log(temp);
-            var sample = OpenBCISample.convertPacketToSample(sampleBuf);
-            assert(sample.channelData[1], -8323071 * OpenBCISample.scaleFactorChannel);
+            var sample = openBCISample.convertPacketToSample(temp);
+            sample.channelData[0].should.be.approximately(-8323071 * openBCISample.scaleFactorChannel, 0.001);
```

Comment:
no, sorry, the actual data values. you could do mean(original - convertedOriginal) to see what size of a difference there is. we know that is not perfect because of precision, otherwise we would be able to assert_equal.